src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -177,10 +178,10 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
     }
   }
 
-  private def copyMigrationFiles(metadata: File, migration: File): Try[Unit] = Try {
+  private def copyMigrationFiles(metadata: File, migration: File, fromVault: Boolean): Try[Unit] = Try {
     trace(metadata, migration)
     val filesXmlFile = (metadata / "files.xml").toString()
-    val migrationFiles = Seq("provenance.xml", "dataset.xml", "files.xml", "emd.xml")
+    val migrationFiles = {if (fromVault) Seq("provenance.xml", "dataset.xml", "files.xml") else Seq("provenance.xml", "dataset.xml", "files.xml", "emd.xml")}
```

Comment:
No need to check for the vault or whatever and more flexible when other files get added.
```suggestion
    val migrationFiles = Seq("provenance.xml", "dataset.xml", "files.xml", "emd.xml")
      .filter(n => (metadata /n).exists)
```

Reply:
true, however i'd rather have it explicitly spelled out. Less surprises that way